crates/nu-cmd-dataframe/src/dataframe/lazy/explode.rs

```diff
@@ -0,0 +1,158 @@
+use crate::dataframe::values::{Column, NuDataFrame, NuExpression, NuLazyFrame};
+
+use nu_protocol::{
+    ast::Call,
+    engine::{Command, EngineState, Stack},
+    Category, Example, PipelineData, ShellError, Signature, Span, SyntaxShape, Type, Value,
+};
+
+#[derive(Clone)]
+pub struct LazyExplode;
+
+impl Command for LazyExplode {
+    fn name(&self) -> &str {
+        "dfr explode"
+    }
+
+    fn usage(&self) -> &str {
+        "Explodes a dataframe or creates a explode expression."
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build(self.name())
+            .rest(
+                "columns",
+                SyntaxShape::String,
+                "columns to explode, only applicable for dataframes",
+            )
+            .input_output_types(vec![
+                (
+                    Type::Custom("expression".into()),
+                    Type::Custom("expression".into()),
+                ),
+                (
+                    Type::Custom("dataframe".into()),
+                    Type::Custom("dataframe".into()),
+                ),
+            ])
+            .category(Category::Custom("lazyframe".into()))
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![
+            Example {
+                description: "Explode the specified dataframe",
+                example: "[[id name hobbies]; [1 Mercy [Cycling Knitting]] [2 Bob [Skiing Football]]] | dfr into-df | dfr explode hobbies | dfr collect",
+                result: Some(
+                   NuDataFrame::try_from_columns(vec![
+                    Column::new(
+                        "id".to_string(), 
+                        vec![
+                            Value::test_int(1),
+                            Value::test_int(1),
+                            Value::test_int(2),
+                            Value::test_int(2),
+                        ]),
+                    Column::new(
+                        "name".to_string(), 
+                        vec![
+                            Value::test_string("Mercy"),
+                            Value::test_string("Mercy"),
+                            Value::test_string("Bob"),
+                            Value::test_string("Bob"),
+                        ]),
+                    Column::new(
+                        "hobbies".to_string(), 
+                        vec![
+                            Value::test_string("Cycling"),
+                            Value::test_string("Knitting"),
+                            Value::test_string("Skiing"),
+                            Value::test_string("Football"),
+                        ]),
+                   ]).expect("simple df for test should not fail")
+                   .into_value(Span::test_data()),
+                    )
+            },
+            Example {
+                description: "todo expression case",
```

Comment:
Can we have a better name here other than `todo`?

Reply:
I forgot to remove after I added the Example.. fixing.